docker-compose.apps.yml

```
@@ -1,51 +1,103 @@
 services:
-  discovery:
-    image: oakcan/discovery:0.0.1-SNAPSHOT
+  discovery-server:
+    image: oakcan/discovery-server:0.0.1-SNAPSHOT
     container_name: oakcan-discovery-server
+    environment:
+      - SPRING_CLOUD_CONFIG_URI=http://config-server:8071
     ports:
       - "8761:8761"
+    healthcheck:
+      test: [ "CMD", "wget", "-qO-", "http://localhost:8761/actuator/health" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)